### ISSUE TITLE
Accessibility: Colorblindness Palettes

### ISSUE THREAD

**Issue by danarchen** [hackforla/311-data#808]
### Dependency
- Waiting on this https://github.com/hackforla/311-data/issues/1157.

### Overview
We need to add a user toggle selection feature to the website so that users can select color pallets that meet their accessibility requirements

- Red / Green
- Blue / Yellow

### Action Items

- [x] Choose colors
- [x] Get [colors and palettes](https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=1%3A5) from UX - Request type color issue already created  
- [ ] Create issue for backend to make palette options available to frontend
   - Make Issue for backend, request data, names, abbreviations (stored). To extend color options, backend needs small changes and frontend creates controls. 
- [ ] Create frontend issue to create control/ toggle 
- [ ] Create issue for user testing once implemented

**Comment by danarchen** [comment]
@kevindperez provided new colors on Figma

**Comment by EchoProject** [comment]
Adding this Issue to items for review for the product roadmap.

**Comment by EchoProject** [comment]
Hi @elizabeth-ux I wanted to see if this issue was still relevant to the v2 launch. It looks like there was some discussion and work done, but there are no additional notes/ comments on what happened next. Could you share what you know regarding the accessibility color palette?

**Comment by EchoProject** [comment]
Make Issue for backend, request data, names, abbreviations (stored). To extend color options, backend needs small changes and frontend creates controls. 

- [ ] Get colors and palettes from UX - Request type color issue already created 
- [ ] Update backend (create new issue)
- [ ] Update frontend (control/toggle)

Wait until colors are implemented and send for user testing.
